app/api/business/agency_business.py

```diff
@@ -30,7 +30,6 @@ def update(agency_id, agency, updated_by):
 
     if agency.get('reports', None) is not None:
         existing.reports = agency.get('reports')
-
     if agency.get('must_join_team', None) is not None:
```

Comment:
Let's leave this whitespace.
```suggestion

    if agency.get('must_join_team', None) is not None:
```